generators/generator-base-private.js

```diff
@@ -987,10 +1025,6 @@ module.exports = class JHipsterBasePrivateGenerator extends Generator {
      */
     generateEntityClientFields(pkType, fields, relationships, dto, customDateType = 'dayjs.Dayjs', embedded = false) {
         const variablesWithTypes = [];
-        const tsKeyType = this.getTypescriptKeyType(pkType);
-        if (!embedded && this.jhipsterConfig.clientFramework !== ANGULAR) {
-            variablesWithTypes.push(`id?: ${tsKeyType}`);
-        }
```

Comment:
Are you going to implement composite id for other react and vue?
If not you should keep this.

Reply:
Same